webpack.config.js

```diff
@@ -0,0 +1,40 @@
+const path = require('path');
```

Comment:
Have you tried the template that serverless proposed for TS Node projects? With esbuild configuration instead of webpack?

Reply:
Done